registry.json

```diff
@@ -3957,5 +3957,19 @@
     },
     "telegram": "Tom_Nimue",
     "operatorName": "Tom"
+  },
+  {
+    "ticker": "ZPCT",
+    "poolId": "d04e478446390de60b5905a0e29a57bc562c936c3620b1a1763cf0ef",
+    "social": {
+      "website": "https://zeropercentstaking.com",
+      "twitter": "",
+      "facebook": "https://www.facebook.com/zeropercentstaking",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "",
```

Comment:
The rules of the alliance are that you must have a TG id. Please add a telegram ID of an individual and not a group or channel, thanks.